### ISSUE TITLE
Update Configuration from Radio Button Attribute only after unfocus

### ISSUE THREAD

**Issue by schuppell** [SAP/spartacus#8812]
Implement custom handler for selection and only update the configuration (keyboard only) when the focus goes away from the radio button group.

**Comment by steinsebastian** [comment]
Current Implementation has a shortcoming if the user switches between keyboard and mouse: After changing a value with the keyboard and then using the mouse to change the group the first click in the group-menu/on previous/next-buttons doesn't work as the focusout event triggers the submit of the value which sets the group to the previous group again.
This has been discussed with Jutta: For the MVP we accept this. It will be discussed with Ux.
To solve this the event have to be queued, which would mean to implement major changes in the ngrx-store.
